datasetinsights/commands/evaluate.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+import click
+
+import datasetinsights.constants as const
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    help="Start model evaluation tasks for a pre-trained model.",
+    context_settings=const.CONTEXT_SETTINGS,
+)
+@click.option(
+    "-c",
+    "--config",
+    type=click.STRING,
```

Comment:
Can be changed to `click.File()`

Reply:
`click.File()` will open the file by default which is usually less desirable. Also, we might want to support this config file from public http url (such as github repo) to allow users passing config files to docker container (instead of having these files that was included during the build process. 

I think it's better to keep it as is for now.

Reply:
I see that it might make more sense to make it a custom ParamType for the future